contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,356 @@
+# Proposal to Standardize Development Process
+
+## Metadata
```

Comment:
Metadata has been pulled into a separate proposal, which is currently a doc:
https://docs.google.com/document/d/1ynmBMuDuT7yGzRscObB1KtgJj8ljYq0I5q4oshrJUCs/edit#heading=h.pmywgz3hu3ii

Reply:
I think the metadata proposal is quite close to done, btw.